.github/workflows/build.yaml

```
@@ -53,50 +55,156 @@ jobs:
+          push: true
+          tags: |
+            xiilab/astrago-keycloak-theme:${{ steps.version.outputs.version }}
+
+  update_astrago_deployment:
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: build
+    name: Update astrago-deployment files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout astrago-deployment
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)